tests/visualizations_tests/test_vis_image.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+
+import numpy as np
+
+from chainer import testing
+
+from chainercv.visualizations import vis_image
+
+try:
+    import matplotlib  # NOQA
+    optional_modules = True
+except ImportError:
+    optional_modules = False
+
+
+class TestVisImage(unittest.TestCase):
+
+    def test_vis_image(self):
+        if optional_modules:
+            img = np.random.uniform(size=(3, 32, 32)).astype(np.float32)
```

Comment:
I think it is better to use `np.random.randint(0, 255), size=...).astype(np.float32)` because the range `vis_image` takes is  `[0, 255]`.